### PR TITLE
Minor fix for web storage

### PR DESCRIPTION
**Pull request by plehegar**

### CODE REVIEW

File: webstorage/idlharness.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <link rel="help" href="http://www.w3.org/TR/webstorage/#storage"/>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/webidl2/lib/webidl2.js"></script>
```
Comment (zqzhang) [inline webstorage/idlharness.html]: Hmm, seems the current being http://w3c-test.org/ doesn't use `WebIDLParser.js` name; shall we update all idlharness.js based test files?

Comment (plehegar) [inline webstorage/idlharness.html]: there is a redirect in wptserve so we shouldn't have to touch anything the idlharness tests. my mistake.

Comment (zqzhang) [inline webstorage/idlharness.html]: Seems so. I would like to revert this change.
